src/core/load_balancing/weighted_round_robin/weighted_round_robin.cc

```diff
@@ -86,6 +88,41 @@ namespace {
 
 constexpr absl::string_view kWeightedRoundRobin = "weighted_round_robin";
 
+constexpr absl::string_view kMetricLabelLocality = "grpc.lb.locality";
+
+const auto kMetricRrFallback =
+    GlobalInstrumentsRegistry::RegisterUInt64Counter(
+        "grpc.lb.wrr.rr_fallback",
+        "EXPERIMENTAL.  Number of scheduler updates in which there were not "
+        "enough endpoints with valid weight, which caused the WRR policy to "
+        "fall back to RR behavior.",
+        "{updates}", {kMetricLabelTarget}, {kMetricLabelLocality}, false);
+
+const auto kMetricEndpointWeightNotYetUsable =
+    GlobalInstrumentsRegistry::RegisterUInt64Counter(
+        "grpc.lb.wrr.endpoint_weight_not_yet_usable",
+        "EXPERIMENTAL.  Number of endpoints from each scheduler update that "
+        "don't yet have usable weight information (i.e., either the load "
+        "report has not yet been received, or it is within the blackout "
+        "period).",
+        "{endpoints}", {kMetricLabelTarget}, {kMetricLabelLocality}, false);
+
+const auto kMetricEndpointWeightStale =
+    GlobalInstrumentsRegistry::RegisterUInt64Counter(
+        "grpc.lb.wrr.endpoint_weight_stale",
+        "EXPERIMENTAL.  Number of endpoints from each scheduler update whose "
+        "latest weight is older than the expiration period.",
+        "{endpoints}", {kMetricLabelTarget}, {kMetricLabelLocality}, false);
```

Comment:
let's follow the same convention here as opentelemetry https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units . s/{endpoints}/{endpoint}

Reply:
Done.